config/routes.rb

```diff
@@ -25,6 +24,11 @@
   }
 
   resources :proposals, only: [:index, :show] do
+    member do
+      get 'approve'
```

Comment:
Per our [Slack convo](https://18f.slack.com/archives/cap/p1430921797004017), not going to worry about this for now, since form support in HTML emails isn't available everywhere.